style.css

```
@@ -384,4 +384,77 @@
   .im {
     margin-bottom: -20px;
   }
+ /*contact form section*/
+ .formsection{
+  background-image: url("./img/header-shapes/ mobile.png");
+  background: #6070FF;
```

```suggestion
  background-color: #6070FF;
  border-top-left-radius: 90px;
```

I see you are trying to use the background as a whole as one image. Try using `background` tag in the container for an easier time with it